pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -0,0 +1,184 @@
+package useraccount
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"github.com/codeready-toolchain/member-operator/pkg/apis"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+)
+
+func TestEnsureUser(t *testing.T) {
```

Comment:
I would be really thankful if we could follow and use the `//given //when //then` parts approach - it really helps with understanding the test